src/MemUtils_CUDA.cxx

```diff
@@ -73,16 +74,30 @@ namespace RAJA
 // Static array used to keep track of which unique ids
 // for CUDA reduction objects are used and which are not.
 //
-static bool cuda_reduction_id_used[RAJA_MAX_REDUCE_VARS];
+static bool s_cuda_reduction_id_used[RAJA_MAX_REDUCE_VARS];
 
 //
 // Pointer to hold shared managed memory block for RAJA-Cuda reductions.
 //
-CudaReductionBlockDataType* s_cuda_reduction_mem_block = 0;
+static CudaReductionDummyBlockType* s_cuda_reduction_mem_block = 0;
```

Comment:
Why are we making these, and others below, static? If it is to ensure they are file scope, then I believe the more common C++ practice for this is to include them in an unnamed namespace.